plugins/aws-s3-storage/README.md

```diff
@@ -56,6 +56,7 @@ store:
     s3ForcePathStyle: false # optional, will use path style URLs for S3 objects
     accessKeyId: your-access-key-id # optional, aws accessKeyId for private S3 bucket
     secretAccessKey: your-secret-access-key # optional, aws secretAccessKey for private S3 bucket
+    sessionToken: your-session-token # optional, aws sessionToken for private S3 bucket
```

Comment:
quick question here: aren't sessionTokens temporary? are there any Docs on their TTL? if they are expired say after one day, does not make sense to put them in config no?

Reply:
Right, sessionTokens are temporary. I think it is okay to have them in the config and users can provide an environment variable instead of hard coding and changing that again, right?

Reply:
well depends on the TTL, would be nice to know in what kind of circumstance they would expire. If it happens too frequently, it would not be a good choice, it would mean, the devOps people have to keep manually update it

Reply:
In our use case, we are not using verdaccio as a long running service. We just start it, connect to it and download packages during build processes and exit. During the build process we use aws roles so this is not a problem. 

However, when we want to push packages from local machines, we only have temporary credentials (token) that expire every day. So, for us to publish packages to S3 using verdaccio from local machines, we need this (unless I am missing something). I believe there will be people who would run into the same. This is not for DevOps, this is for developers essentially.

Reply:
oh interesting use case. We went a different way, having verdaccio as a running service on an aws instance. In any case, it is maybe worth mentioning that in the Docs? @juanpicado I wonder if there is specific Docs, on best practices when it comes to verdaccio (as long running service, or one-off Dev as mentioned).

Reply:
First time I heard that his plugin is being used for merely development purposes, which is cool :) . 

As @tetrohed suggests, I'd update the readme, unfortunately, we do not have docs in our website regarding specific plugins, the main reason is (at least me) don't have much domain of AWS itself so I guess users will come to read the README (it's what I'd do). 

Perhaps the future website might have more info about each plugin (https://github.com/verdaccio/website_2.0) like GatsbyJS does for instance, but not the case at this moment.

@stackdumper feel free to update the README (add a new section) and tell more about your use case and the new property you are adding, I guess others will find your experience really interesting.